$MirOS: ports/games/frozen-bubble/patches/patch-server_net_c,v 1.2 2006/11/04 04:12:04 tg Exp $
--- server/net.c.orig	Thu Nov 23 21:44:26 2006
+++ server/net.c	Sat Dec 23 01:24:39 2006
@@ -49,6 +49,10 @@
 #include "log.h"
 #include "net.h"
 
+#ifndef MSG_NOSIGNAL
+#define MSG_NOSIGNAL 0
+#endif
+
 /* this is set in game.c and used here for answering with
  * the requested command without passing this additional arg */
 char* current_command;
